src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,195 @@
+    
+  </Step>
+
+  <Step>
+
+    ## Add users
+
+    You can add the rest of your team to New Relic. Use the **User management** UI to [add users](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks#where). See [User type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type) to learn more about the different types of users. You can also check [our tutorial](/docs/accounts/accounts-billing/new-relic-one-user-management/account-user-mgmt-tutorial) on adding users and managing user access.
+  </Step>
+</Steps>
+
+
+
+# Capabilities [#capabities]
+
+Our platform offers a comprehensive suite of capabilities to help organizations monitor, troubleshoot, and optimize the performance of their digital environments. Here's an explanation of New Relic's key capabilities. If you want to learn more, explore our [tutorials](/docs/new-relic-solutions/tutorial-landing-page/) pages. 
+
+
+## Alerts & AI [#alerts]
+
+Alerts help you identify issues in your applications and code. They show you how these issues affect your users. When you receive notifications of your alerts, you can stop problems and prevent widespread user disruptions. Being aware of issues early can lead to better user experiences.
+
+Let us help you know what's critical, manage the suppressed noise, and mitigate alert fatigue.
+
+<img
+  title="New Relic alerts main page"
+  alt="New Relic alerts main page"
+  src={NRAlertsMain}
+/>
+
+
+
+## APM & Services [#apm]
+
+New Relic instruments your application at the code level through the use of one of our many language agents. These agents collect metrics from your application and send them to our application performance monitoring (APM) , allowing you to monitor it using pre-built dashboards.
+
+The <InlinePopover type="apm" /> capability provides real-time insights into the performance of applications, services, and microservices across various programming languages and frameworks.
+
+It offers detailed transaction traces, code-level visibility, and performance metrics to identify bottlenecks, optimize application performance, and enhance user experience.
+
+<img
+  title="New Relic APM & Services main page"
+  alt="New Relic APM & Services main page"
+  src={NRApmMain}
+/>
+
+
+## Browser [#browser]
+
+The <InlinePopover type="browser" /> capability provides insights into the performance and user experience of your web applications, directly within users' browsers.
+
+
+<img
+  title="New Relic Apps main page"
+  alt="New Relic Apps main page"
+  src={NRBrowserMain}
+/>
+
+
+## Dashboards [#dashboards]
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. In ultricies ornare quam, a elementum tellus vestibulum et. Donec malesuada cursus pellentesque. Sed finibus elementum lorem quis efficitur. Donec blandit facilisis aliquet. Integer vel suscipit augue. Quisque a metus gravida, consequat arcu in, auctor enim. Morbi libero quam, dictum vel ornare ut, dignissim eu nisi.
```

Comment:
Always love to see the lorem ipsum :)

Probably a good place to throw in a picture of chart.

Dashboards by themselves don't do much. They're kind of a starting place and they make for good visuals. I think the important thing to emphasize about dashboards is that they're portals into the very cool and powerful data analysis tools that New Relic provides.